scss/bitstyles.scss

```diff
@@ -21,70 +21,69 @@
 //
 // Atoms //////////////////////////////////////
 
-@import 'bitstyles/atoms/list-reset/';
-@import 'bitstyles/atoms/icon/';
+@import 'bitstyles/atoms/avatar/';
```

Comment:
Hm, it probably makes sense to add a rule for `stylelint` or `eslint` to make sure this is also adhered to after this PR is merged 😉

Reply:
Ooo, great idea 💡✨I’ll add a ticket to look into that

Reply:
#460